backend/inspirehep/records/api/base.py

```diff
@@ -553,3 +553,10 @@ def redirected_record_ref(self):
         if self.get("deleted") and pid.status == PIDStatus.REDIRECTED:
             redirection = InspireRedirect.get_redirect(pid)
             return get_ref_from_pid(redirection.pid_type, redirection.pid_value)
+
+    def check_if_record_was_deleted_by_merge(self):
```

Comment:
don't we already have a similar function that is used to decide whether to perform the merge?